### DISCUSSION TITLE
عدم بلاک سایت‌های ایرانی

### DISCUSSION THREAD

**Discussion by seapou** [hiddify/Hiddify-Manager#942] [Q&A سوالات رایج]
با عرض تشکر و خسته نباشید بخاطر وقت و انرژی و توانی که میذارید برای عبور از فیلترینگ، میخواستم بگم حداقل برای من وقتی گزینه بلاک کردن سایت‌های ایرانی فعاله کلا هیچ سایت ایرانی رو باز نمیکنه.

**Comment by h9ee** [comment]
برای من وقتی کانکت هستم و سایت های فیلتر شده رو باز نمی کنه اینجور می شه ! که به این منظوره اصلا کانکت نیستید

**Reply by seapou** [reply]
نه متصلم سایت های خارجی اعم از فیلتر و غیر فیلتر رو باز میکنه

**Reply by h9ee** [reply]
اینا باگ های هست که سخت می شه فهمید از کجاست! اما من چیزی که متوجه شدم از سرور جدید که گرفتم! خوده سرور و ای پی که می ده هستش !
من مشکلاتی تو سرور جدید دارم که تو قبلی نداشتم ! و برعکس ! 

اگر تونستی ای پی سرور رو عوض کن!‌
